gce/setup.sh

```diff
@@ -17,6 +17,10 @@ helm repo update
 # Install JupyterHub and Dask on the cluster
 helm install jupyterhub/jupyterhub --version=v0.6.0-9701a90 --name=jupyter --namespace=pangeo -f jupyter-config.yaml -f secret-config.yaml
 
+# create the daskkubernetes service account and role bindings
+echo "Installing service account for daskkubernetes."
+kubectl create -f dask-kubernetes-serviceaccount.yaml
```

Comment:
TL;DR This is an anti-pattern, but probably fine for now.

I'm slightly uncomfortable having additional manifests being added manually as you no longer simply run `helm delete <release> --purge` to remove pangeo. It would be better to have the whole deployment contained in a single helm installation. This could be a good opportunity to create a `pangeo` helm chart.

We currently have a `jadepangeo` [helm chart](https://github.com/met-office-lab/jade-pangeo) which has `jupyterhub/jupyterhub` as a dependency and adds some extra manifests. I would like to get to a position where there is an offical chart which depends on jupyterhub and then we can make `jadepangeo` depend on `pangeo/pangeo`. However this is beyond the scope of this PR so I'll raise a new issue for start work on a PR.

Reply:
See #178